src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -103,7 +103,13 @@ export class GraphStyles {
           }
 
           if (rate > 0) {
-            content = pErr > 0 ? rate.toFixed(2) + ', ' + pErr.toFixed(1) + '%' : rate.toFixed(2);
+            if (pErr > 0) {
+              let sErr = pErr.toFixed(1);
+              sErr = `, ${sErr.endsWith('.0') ? pErr.toFixed(0) : sErr}%`;
+              content = rate.toFixed(2) + sErr;
```

Comment:
Minor -- I think code will look simpler doing scaling:
```js
let sErr = Math.round(pErr * 10) / 10;
content = `${rate.toFixed(2)}, ${sErr.toString()}%`;
```
BUT... I'm not sure if this could fall in [floating point precision problems](https://stackoverflow.com/questions/588004/is-floating-point-math-broken) and display lots of digits. Probably, your version is safer for the UI, but scaling is easier to read for devs.

So, it's up to you.

Reply:
It's working, I'm OK as is.